tests/js/client/shell/api/multi/import-novst.js

```diff
@@ -536,7 +536,7 @@ function import_into_an_edge_collection__using_a_JSON_arraySuite () {
     },
 
     test_JSONA_multiple_docs__with_wrong_fromPrefix: function() {
-      let cmd = api + `?collection=${en}&createCollection=false&type=array&fromPrefix=a+b`;
+      let cmd = api + `?collection=${en}&createCollection=false&type=array&fromPrefix=a/b`;
```

Comment:
Without extended collection names, using a collection `a+b` (URL-decoded this would be `a b`) would be an illegal name.
With the change in this PR and using extended collection names, `a b` becomes a valid name, so the test rightfully failed.
So I made the collection name used in the test illegal again.
But I now pushed a change to the PR that should make it more clear.

Reply:
Thanks